libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/LogHelper.kt

```diff
@@ -1,28 +1,28 @@
 package breez_sdk_notification
 
-import org.tinylog.kotlin.Logger
-import java.io.File
+import breez_sdk.setLogStream
+import kotlinx.coroutines.CoroutineScope
 
 @Suppress("unused")
-class LogHelper {
-    companion object {
-        private const val TAG = "LogHelper"
+object LogHelper {
```

Comment:
Would be great to get rid of the `LogHelper` and the `SdkLogListener` as they are static and also implement subscribe/unsubscribe for multiple listeners which I don' think belong to the sdk level.
How about removing these two classes and Implement a function on the `ForegroundService` `setLogListener` which the user can use.

Reply:
I agree, I think its simpler if the application could set LogStream listeners for the SDK and the notification plugin (add setLogStream to foreground service), then the application can tag the log entries as they want and no changes to sdk-core are needed

Reply:
Logger is now non-static. edf235294e449b3cf241de05971d67a831c221ab